client/reader/full-post/controller.js

```diff
@@ -46,17 +46,18 @@ export function blogPost( context ) {
 	trackPageLoad( basePath, fullPageTitle, 'full_post' );
 
 	ReactDom.render(
```

Comment:
would it be better to change this to `renderWithReduxStore` or does it not matter?